src/docker/composebuilder/development.go

```diff
@@ -127,16 +126,11 @@ func (d *DevelopmentDockerComposeBuilder) getExternalServiceDockerConfigs() (typ
 	if d.Mode.Environment == BuildModeEnvironmentTest {
 		return result, nil
 	}
-	renderedVolumes, err := tools.GetRenderedVolumes(d.ServiceConfig.Docker.Volumes, d.AppConfig.Name, d.Role, d.HomeDir)
```

Comment:
I don't think we were really using this before, as the `ServiceConfig.Docker` block has been removed anyways. We can add volume support to external images back in as needed.